### PR TITLE
update connect transfer function

### PR DESCRIPTION
**Pull request by daiki44**
- add transfer function

### CODE REVIEW

File: src/components/Send.js [hunk 1]
```diff
@@ -52,21 +54,36 @@ const Send = props => {
...
         full
-        onClick={e => {
-          props.setTransferredAmount(tokenAmount)
-          props.setRecepientAddress(recepientAddressRef.current.value)
-          // const href = `${router.route}?transfer`
-          // router.push(href, href, { shallow: true })
+        onClick={() => {
+          setIsLoading(true)
+          props
+            .transfer(
```
Comment (daiki44) [inline src/components/Send.js:70]: How do I do validation checks and clearing state? What's the best practice to do it?

File: src/components/Send.js [hunk 2]
```diff
@@ -52,21 +54,36 @@ const Send = props => {
...
+              props.recepientAddress
+            )
+            .then(() => {
+              setIsLoading(false)
+              props.setTransferredAmount('')
+              props.setRecepientAddress('')
+            })
         }}
-        disabled
+        disabled={
```
Comment (daiki44) [inline src/components/Send.js:81]: How do I check the button disable or not? Is this way correct?

Comment (tkmct) [reply src/components/Send.js:81]: Please use selector.

File: src/components/Send.js (hunk 1 above)
Comment (tkmct) [reply src/components/Send.js:70]: please write these logic in thunk.

File: src/components/Send.js [hunk 3]
```diff
@@ -52,21 +54,36 @@ const Send = props => {
       <TokenInput
         className="mts mbs"
         unit={transferredTokenObj.unit}
-        balance={roundBalance(props.ETHtoUSD, tokenAmount)}
-        onChange={e => {
-          setTokenAmount(e.target.value)
-        }}
+        balance={roundBalance(props.ETHtoUSD, props.transferredAmount)}
+        handleAmount={inputAmount => props.setTransferredAmount(inputAmount)}
```
Comment (tkmct) [inline src/components/Send.js:58]: you can write just`handleAmount={props.setTransferredAmount}`

File: src/components/Send.js [hunk 4]
```diff
@@ -52,21 +54,36 @@ const Send = props => {
...
+      />
+      <AddressInput
+        className="mbs"
+        type="text"
+        handleAddress={inputAddress => props.setRecepientAddress(inputAddress)}
```
Comment (tkmct) [inline src/components/Send.js:63]: same here

File: src/store/transfer.js [hunk 5]
```diff
@@ -48,6 +48,7 @@ export const transfer = (amount, depositContractAddress, recipientAddress) => {
       dispatch(setTransferPage('completion-page'))
     } catch (error) {
       console.log(error)
+      alert(error)
```
Comment (tkmct) [inline src/store/transfer.js:51]: Please don't use `alert` unless for debugging purpose.

**Comment by daiki44** [comment]
Error message image
スクリーンショット 2020-05-14 10 46 01

**Comment by daiki44** [comment]
@tkmct updated!

**Pull request merged**